07-regression1.Rmd

```diff
@@ -84,21 +95,22 @@ is fair, or perhaps how to set the price of a new listing.
 
 The scientific question guides our initial exploration: the columns in the
 data that we are interested in are `sqft` (house size, in livable square feet)
-and `price` (house price, in US dollars (USD)).  The first step is to visualize
+and `price` (house sale price, in US dollars (USD)).  The first step is to visualize
 the data as a scatter plot where we place the predictor/explanatory variable
 (house size) on the x-axis, and we place the target/response variable that we
-want to predict (price) on the y-axis:
+want to predict (sale price) on the y-axis.
 
-```{r 07-edaRegr, message = FALSE, fig.height = 4, fig.width = 5, fig.cap = "Scatter plot of price (USD) versus house size (square footage)"}
+```{r 07-edaRegr, message = FALSE, fig.height = 4, fig.width = 5, fig.cap = "Scatter plot of price (USD) versus house size (square feet)"}
 eda <- ggplot(sacramento, aes(x = sqft, y = price)) +
   geom_point(alpha = 0.4) +
-  xlab("House size (square footage)") +
+  xlab("House size (square feet)") +
   ylab("Price (USD)") +
   scale_y_continuous(labels = dollar_format())
 eda
 ```
-
-Based on the visualization above, we can see that in Sacramento, CA, as the
+ 
+The plot is shown in Figure \@ref(fig:07-edaRegr).
+We can see that in Sacramento, CA, as the
 size of a house increases, so does its sale price. Thus, we can reason that we
 may be able to use the size of a not-yet-sold house (for which we don't know
 the sale price) to predict its final sale price. Note that we do not suggest here
```

Comment:
Should we go a step further here and start to say more why correlation or prediction != causation? Or at least define causation and contrast it with prediction?

Reply:
no, i don't think we should get into it. too nuanced. if you feel strongly we *could* add something to the additional resources section at the end of the regression section about resources to seek for causal stuff.